dbfit-java/build.gradle

```diff
@@ -55,6 +55,25 @@ allprojects {
     dependencies {
         compile "org.fitnesse:fitnesse:${fitNesseVersion}:standalone@jar"
     }
+    
+    // fix gradle .classpath generation for bin folders and put it under /build/eclipse
+	eclipse {
+		classpath {
+            defaultOutputDir = file('build/eclipse') // this should be enough, but it's not
+		    file.whenMerged {
+			   entries.each { entry ->
+					if (entry instanceof org.gradle.plugins.ide.eclipse.model.Output) {
+					   entry.path = entry.path.replace('bin/main', 'build/eclipse/main')
```

Comment:
Aren't output entries already OK?

Reply:
No, and I don't know why this was not a problem before. May be gradle changed behavior here and Eclipse has not been used by somebody for some time. I think most professional Java Developers use IDEA and thus this may not have been a problem. What I see is that gradle changed the output dir from "bin" to "bin/[main|test|integrationTest]" with the upgrade to 6.6.